examples/get_alerts.rb

```diff
@@ -0,0 +1,10 @@
+require 'prometheus/alert_client'
+
+# returns a client
+prometheus = Prometheus::AlertClient.client(url: 'http://localhost:9099', path: '/topics/zgalor')
+
+alerts = prometheus.get('12497ca8-b597-4590-ac5d-d55af7f3d185', 2)
```

Comment:
Since kw arguments are great for sensibility, consider making all the exposed methods use them